.github/workflows/run-tests.sh

```diff
@@ -43,5 +43,56 @@ if [ $RUNTIME = "core" ] || [ $RUNTIME = "knative" ]; then
 
 elif [ $RUNTIME = "streaming" ]; then
   echo "TODO: test the streaming runtime"
+  # TODO move setup to fats
+  ${FATS_DIR}/install.sh helm
+  source ${FATS_DIR}/macros/helm-init.sh
+
+  helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+  helm install --name my-kafka incubator/kafka --set replicas=1,zookeeper.replicaCount=1,zookeeper.env.ZK_HEAP_SIZE=128m --namespace $NAMESPACE --wait
+
+  riff streaming kafka-provider create franz --bootstrap-servers my-kafka:9092 --namespace $NAMESPACE
+
+  cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: Pod
+metadata:
+  name: dev-utils
+spec:
```

Comment:
avoid granting extra permission to the default service account. We wouldn't want uses to do that, and we don't want to give our workloads extra privileges they shouldn't have.

Accepting this suggestion will also necessitate, creating the service account and updating the role binding.

```suggestion
spec:
  serviceAccountName: dev-utils
```